docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml

```
@@ -0,0 +1,484 @@
+  standards and process standards developed, maintained and interpreted by USDA’s Agricultural
+  Marketing Service.
+
+  (source: USDA Agricultural Marketing Service: Grades and Standards,
+  https://www.ams.usda.gov/grades-standards).
+type: object
+properties:
+  '@context':
+    type: array
+    items:
+      const:
+        - 'https://www.w3.org/2018/credentials/v1'
+        - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    items:
+      const:
+        - VerifiableCredential
+        - FoodGradeInspectionCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    $ref: ../common/FoodGradeInspection.yml
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ../common/LinkRole.yml
+additionalProperties: false
+required: []
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "https://example.com/credential/123",
+    "type": [
+      "VerifiableCredential",
+      "FoodGradeInspectionCertificate"
+    ],
+    "name": "Food Grade Inspection Certificate",
+    "relatedLink": [
+      {
+        "type": [
+          "LinkRole"
+        ],
+        "target": "https://www.ams.usda.gov/grades-standards",
+        "linkRelationship": "usdaAmsGradesAndStandardsLink"
+      }
+    ],
+    "issuanceDate": "2021-12-11T03:50:55Z",
+    "issuer": {
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "type": "Organization",
+      "name": "Hauck Group",
+      "description": "Focused secondary synergy",
+      "address": {
+        "type": "PostalAddress",
+        "streetAddress": "3198 O'Kon Wall",
+        "addressLocality": "North Adolphchester",
+        "addressRegion": "Nevada",
+        "postalCode": "50788",
+        "addressCountry": "Cote d'Ivoire"
+      },
+      "email": "Bernita.Quitzon98@example.com",
+      "phoneNumber": "555-171-4411",
+      "faxNumber": "+1-555-758-9761"
+    },
+    "credentialSubject": {
+      "type": "FoodGradeInspection",
+      "facility": {
+        "type": [
+          "Place"
+        ],
+        "globalLocationNumber": "5449782976823",
+        "geo": {
+          "type": [
+            "GeoCoordinates"
+          ],
+          "latitude": "-79.6395",
+          "longitude": "178.5353"
+        },
+        "address": {
+          "type": [
+            "PostalAddress"
+          ],
+          "organizationName": "Ace Foodstuffs",
+          "streetAddress": "853 Wisozk River",
+          "addressLocality": "New Noemyfort",
+          "addressRegion": "New Mexico",
+          "postalCode": "18047-2038",
+          "addressCountry": "Togo"
```

New Mexico is in the USA, and that's a US ZIP+4 Code... 
```suggestion
          "addressRegion": "New Mexico",
          "postalCode": "18047-2038",
          "addressCountry": "USA"
```

---

Sure - these are generated automatically but happy to change it

---

@rhofvendahl -- Ah, then whatever's generating these sample values needs some attention. We do want fake addresses, etc., for sample data (so we run no risk of causing trouble for real entities), but it should be usable with, for instance, tools that display such entities on real maps, tools that calculate distances between them, etc., for evaluation of both those tools and the traceability vocab. /cc @mprorock @OR13